kong/clustering/utils.lua

```diff
@@ -164,4 +165,33 @@ function _M.is_dp_worker_process()
   return worker_id() == 0
 end
 
+
+-- encode/decode json with cjson or simdjson
+local ok, simdjson_dec = pcall(require, "resty.simdjson.decoder")
+if not ok or kong.configuration.cluster_cjson then
```

Comment:
Should we give `cluster_cjson` a default value?

If `not ok`, and its default value is `nil`, this code forces `resty.simdjson`.

Reply:
added, thanks.